UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/wwwroot/css/nm_style.css

```diff
@@ -0,0 +1,238 @@
+﻿.site-header .navbar {
```

Comment:
Is `nm` in file name **nm**_style.css means Notice to Mariners (NM)? If yes then NM is a feature/area in the application and I believe css file will be common for entire application. Please can we rename css file name to be app specific.

Reply:
rename scss file as suggested.